packages/dnd/src/drop-cursor/drop-cursor.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { createRef, Fragment, useCallback, useEffect, useRef } from "react";
+
+import findTargetElement from "../find-target-element";
+import useDispatch from "../use-dispatch";
+import useState from "../use-state";
+
+const DropCursor: React.FC<{ CursorProps?: any }> = ({ CursorProps }) => {
+    const selectorRef = createRef<HTMLDivElement>();
+    const mouseRef = createRef<HTMLDivElement>();
+    const targetContainer = useRef<any>();
+
+    const state = useState();
+    const dispatch = useDispatch();
+
+    const handleMouseMove = React.useCallback(
+        (e: any) => {
+            if (!selectorRef.current || !mouseRef.current) {
+                return;
+            }
+            selectorRef.current.style.border = '';
+            selectorRef.current.style.outline = '';
+
+            let cursorPosition = { x: e.x, y: e.y }
+            if (e.type === 'touchmove') {
+                cursorPosition = {
+                    x: e.touches[0].clientX,
+                    y: e.touches[0].clientY,
+                }
+            }
+
+            const { rect, container, position } = findTargetElement(cursorPosition, state);
+
+            if (rect) {
+                selectorRef.current.style.top = `${rect.top}px`;
+                selectorRef.current.style.left = `${rect.left}px`;
+                selectorRef.current.style.width = `${rect.width - 8}px`;
+                selectorRef.current.style.height = `${rect.height}px`;
+                selectorRef.current.style.border = '3px solid #A40000';
```

Comment:
We might want to allow customization or add classes to the elements instead of inline styles to enable visual customization

Reply:
Let's keep that for the future https://github.com/data-driven-forms/editor/issues/10